setup.py

```diff
@@ -53,6 +53,8 @@
     "jupysql-plugin",
     # for monitoring access to files
     "psutil",
+    # for running tests for %sqlcmd connect
+    "jupyter-server",
```

Comment:
do we still need this? I think it was used in the old tests, right? doesn't look like it's used anymore

Reply:
```
Run nox --session test_unit --no-install --reuse-existing-virtualenvs
  
  nox --session test_unit --no-install --reuse-existing-virtualenvs
  shell: /usr/bin/bash -e {0}
  env:
    PLOOMBER_VERSION_CHECK_DISABLED: true
    PYTHON_VERSION: 3.11
    pythonLocation: /opt/hostedtoolcache/Python/3.11.4/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.11.4/x64/lib
nox > Running session test_unit
nox > Re-using existing virtual environment at .nox/test_unit.
nox > python -c 'import sqlalchemy; assert int(sqlalchemy.__version__.split('"'"'.'"'"')[0]) == [2](https://github.com/ploomber/jupysql/actions/runs/5897893433/job/15998164223?pr=783#step:6:2)'
nox > pytest src/tests/ --ignore src/tests/integration
============================= test session starts ==============================
platform linux -- Python [3](https://github.com/ploomber/jupysql/actions/runs/5897893433/job/15998164223?pr=783#step:6:3).11.[4](https://github.com/ploomber/jupysql/actions/runs/5897893433/job/15998164223?pr=783#step:6:4), pytest-7.4.0, pluggy-1.2.0
rootdir: /home/runner/work/jupysql/jupysql
configfile: pyproject.toml
collected 736 items / 1 error

==================================== ERRORS ====================================
_________________ ERROR collecting src/tests/test_magic_cmd.py _________________
ImportError while importing test module '/home/runner/work/jupysql/jupysql/src/tests/test_magic_cmd.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/hostedtoolcache/Python/3.11.4/x64/lib/python3.11/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
src/tests/test_magic_cmd.py:12: in <module>
    from jupysql_plugin.widgets import ConnectorWidget
.nox/test_unit/lib/python3.11/site-packages/jupysql_plugin/__init__.py:2: in <module>
    from .server_handlers.dashboard import setup_handlers
.nox/test_unit/lib/python3.11/site-packages/jupysql_plugin/server_handlers/dashboard.py:3: in <module>
    from jupyter_server.base.handlers import APIHandler
E   ModuleNotFoundError: No module named 'jupyter_server'
=============================== warnings summary ===============================
src/sql/parse.py:193
  /home/runner/work/jupysql/jupysql/src/sql/parse.py:193: DeprecationWarning: invalid escape sequence '\:'
    """

src/sql/_testing.py:29
  /home/runner/work/jupysql/jupysql/src/sql/_testing.py:29: PytestCollectionWarning: cannot collect test class 'TestingShell' because it has a __init__ constructor (from: src/tests/test_testing.py)
    class TestingShell(InteractiveShell):

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
ERROR src/tests/test_magic_cmd.py
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
========================= 2 warnings, 1 error in 1.4[5](https://github.com/ploomber/jupysql/actions/runs/5897893433/job/15998164223?pr=783#step:6:5)s =========================
Closing duckdb://
Closing sqlite://
Closing DuckDBPyConnection
Closing Connection
nox > Command pytest src/tests/ --ignore src/tests/integration failed with exit code 2
nox > Session test_unit failed.
Error: Process completed with exit code 1.
```

Reply:
ok, I'll make a new jupysql-plugin release with this fix so the CI passes.

Reply:
@bbeat2782 I released jupysql-plugin 0.2.0, please try again